tableauserverclient/models/connection_credentials.py

```diff
@@ -9,10 +9,11 @@ class ConnectionCredentials(object):
 
     """
 
-    def __init__(self, name, password, embed=True):
+    def __init__(self, name, password, embed=True, oauth=True):
```

Comment:
I think this should default to 'false' since most connections don't need an oAuth parameter (and I'm not sure what they'll do if both are present)